x-pack/metricbeat/module/aws/aws.go

```diff
@@ -86,7 +87,9 @@ func NewMetricSet(base mb.BaseMetricSet) (*MetricSet, error) {
 	}
 
 	// Get IAM account name
-	svcIam := iam.New(awsConfig)
+	awsConfig.Region = "us-east-1"
```

Comment:
The hardcoded region here works well in global regions but not for China regions.

Reply:
Thank you for the input! Sorry unfortunately I'm not able to test it with a China region, what would this be for China regions? 
With this use case, we probably should add a separate config parameter called `init_region` or something instead of hardcoded this as `us-east-1`. Do you mind creating a separate github issue for this? Thanks again!!!